### ISSUE TITLE
Menu Style conflict when using fullscreen extension

### ISSUE THREAD

**Issue by datune** [FrDH/mmenu-js#1145]
@FrDH 

After ugprading and pulling my hair for 14 hours, I finally found out I have the exact same problem as OP. I have created an updated gist for your perusal using all the latest versions to showcase the problem. 

Basically, as soon as fullscreen is added to one menu, it seems to affect all other menus.

https://gist.github.com/datune/9a4ac9a2e0c42fcbff7e325b44792bc4

_Originally posted by @datune in https://github.com/FrDH/mmenu-js/issues/790#issuecomment-633028245_

**Comment by datune** [comment]
@FrDH I just installed 8.5.11, and it does not appear to be fixed. I still see the same behaviour.

**Comment by datune** [comment]
From what I can tell, as soon as 1 menu has the class mm-menu_fullscreen applied, it affects the slideout, even if that menu is not the one being opened. 

`.mm-wrapper_opening .mm-menu_fullscreen~.mm-slideout {
    -webkit-transform: translate3d(100vw,0,0);
    transform: translate3d(100vw,0,0);
}`

Suppose you have a shopping cart menu defined as:

`    var $cartMenu = $("#cart").mmenu({
        extensions: {
            all: ["position-right"]
            , "(max-width: 451px)": ["fullscreen"],
        },
        navbar: {
            add: false
        }
    });`

alongside a regular, default navigation menu, and you open the default navigation, the slideout will be affected by the settings of the cartMenu, even though it is not opened.

**Comment by datune** [comment]
I wonder, would it not be fixed if you added the mm-menu_opened class, so as to avoid the style being applied to menus that don't actually have mm-menu_fullscreen set?
```
.mm-wrapper_opening .mm-menu_fullscreen.mm-menu_opened~.mm-slideout {
```

**Comment by FrDH** [comment]
That is exactly what i did:
https://github.com/FrDH/mmenu-js/blob/master/src/core/offcanvas/_mixins.scss#L14

But i think it didnt compile right, I'll have another try later on today.

**Comment by datune** [comment]
@FrDH First, thank you. Second, never mind, I'm to blame. I noticed in your commit that this is exactly what you did, so I started looking into the npm package, and I saw the same compiled code there. Than I looked into what my browser served, and after deleting the application cache, it now works. Apologies for any inconvenience! Next time I'm in the Netherlands, I'll have to buy you "een pakje frietjes en een paar pintjes" ;-)
